tequilapi/endpoints/service_sessions.go

```diff
@@ -70,11 +70,7 @@ func NewServiceSessionsEndpoint(sessionStorage serviceSessionStorage) *serviceSe
 //     schema:
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (endpoint *serviceSessionsEndpoint) List(resp http.ResponseWriter, request *http.Request, params httprouter.Params) {
-	sessions, err := endpoint.sessionStorage.GetAll()
-	if err != nil {
-		utils.SendError(resp, err, http.StatusInternalServerError)
```

Comment:
Now that this is removed, is the swagger metadata still relevant?

```
500:
//     description: Internal server error
//     schema:
//       "$ref": "#/definitions/ErrorMessageDTO"
```

Reply:
Good catch